types/ember/index.d.ts

```diff
@@ -107,7 +111,13 @@ declare module 'ember' {
 
     type ObserverMethod<Target, Sender> =
         | (keyof Target)
-        | ((this: Target, sender: Sender, key: keyof Sender, value: any, rev: number) => void);
+        | ((
```

Comment:
changes like these make it very hard to review this PR, it's hard to see what the actual changes are. I would prefer you revert all the extra linebreaks you add in this PR... and if you really care about them make a separate PR with just those changes.